packages/devtools_app/lib/src/debugger/flutter/debugger_controller.dart

```diff
@@ -359,26 +357,50 @@ class DebuggerController extends DisposableController
   }
 
   Future<void> _populateScripts(Isolate isolate) async {
-    _scriptList.value = await _service.getScripts(isolateRef.id);
+    final scriptList = await _service.getScripts(isolateRef.id);
 
     // TODO(devoncarew): Follow up to see why we need to filter out non-unique
     // items here.
-    final scriptRefs = Set.of(_scriptList.value.scripts).toList();
+    final scriptRefs = Set.of(scriptList.scripts).toList();
     scriptRefs.sort((a, b) {
       // We sort uppercase so that items like dart:foo sort before items like
       // dart:_foo.
       return a.uri.toUpperCase().compareTo(b.uri.toUpperCase());
     });
     _sortedScripts.value = scriptRefs;
 
+    // TODO(devoncarew): Switch to calling the typed method once it's in the VM
+    // service protocol library.
+    // ignore: unawaited_futures
+    _service
```

Comment:
Oooh! Can you move this secret method to a shared class? This will clean up an in progress cl for the inspector.

Fyi: @grouma not sure you have implemented this method yet in dwds.

Reply:
Will do.

FYI, Ben's documenting the call here: https://dart-review.googlesource.com/c/sdk/+/144685

Reply:
Moved to `VMServiceWrapper` -

Reply:
I do not believe we implement this call.